src/pages/Fox/components/FoxGovernance.tsx

```diff
@@ -99,6 +100,21 @@ export const FoxGovernance = () => {
     )
   }, [closedProposals])
 
+  const govDescTranslationComponents: TextPropTypes['components'] = useMemo(
+    () => ({
+      link: (
+        <Link
+          colorScheme='blue'
+          color='blue.300'
+          href='https://forum.shapeshift.com/'
+          isExternal
+          mx={1}
```

Comment:
This is causing an unnecessary gap between the link and the text:

<img width="490" alt="Screenshot 2025-03-27 at 5 12 43 pm" src="https://github.com/user-attachments/assets/6f3240ce-86ea-4d91-b983-39a91fd56623" />

```suggestion
```

Reply:
Eagle eye 🦅  Right on!

Reply:
![image](https://github.com/user-attachments/assets/ae79009e-205a-4a5c-ad5d-345de2d5e318)